queries.sql

```diff
@@ -30,3 +30,43 @@ WHERE name <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4  AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species='unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species from animals; 
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT species from animals;
+COMMIT;
+SELECT species from animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] After the DELETE line you need to verify the data with a SELECT statement.  Just like you did in the previous code ☝️ 

Example:
```SQL
DELETE FROM animals;
SELECT COUNT(*) FROM ANIMALS;
```

- [ ] Please, do the same after the ROLLBACK statement. Verify the data with a SELECT statement, as per the requirements of the project.  You now know how to do it 💪